tests/unit/test_charm.py

```diff
@@ -238,59 +240,56 @@ def test_filter_lines(self, test_input, expected):
     @mock.patch("ops.model.Container.get_service")
     @mock.patch("ops.model.Container.make_dir")
     @mock.patch("ops.model.Container.push")
-    @mock.patch("ops.model.Container.start")
-    @mock.patch("ops.model.Container.stop")
+    @mock.patch("ops.model.Container.pebble")
     def test_configure_workload_container_container_not_running(
-        self, stop, start, push, make_dir, get_service, add_layer, make_pebble_config
+        self, pebble, push, make_dir, get_service, add_layer, make_pebble_config
     ):
         """
         arrange: config is changed
         act: check if service is running and is not
-        assert: workload container is stopped
+        assert: services are not replanned
         """
         config = self.config
         harness = self.harness
         harness.update_config(config)
         make_pebble_config.assert_called_once()
         get_service.return_value.is_running.return_value = False
         harness.update_config(config)
-        stop.assert_called_once()
+        pebble.replan_services().assert_not_called()
 
     @mock.patch("charm.ContentCacheCharm._make_pebble_config")
     @mock.patch("ops.model.Container.add_layer")
     @mock.patch("ops.model.Container.get_service")
     @mock.patch("ops.model.Container.make_dir")
     @mock.patch("ops.model.Container.push")
-    @mock.patch("ops.model.Container.start")
-    @mock.patch("ops.model.Container.stop")
+    @mock.patch("ops.model.Container.pebble")
     def test_configure_workload_container_pebble_services_already_configured(
-        self, stop, start, push, make_dir, get_service, add_layer, make_pebble_config
+        self, pebble, push, make_dir, get_service, add_layer, make_pebble_config
     ):
         """
         arrange: config is changed
-        act: check if current config is different and is not
-        assert: no action
+        act: check if current config is different
+        assert: Services are replanned again
```

Comment:
There seems to be capitalisation here that there isn't elsewhere. Can we make that consistent please?